cheetah/accelerator.py

```diff
@@ -2442,3 +2442,143 @@ def __repr__(self) -> str:
             f"{self.__class__.__name__}(elements={repr(self.elements)}, "
             + f"name={repr(self.name)})"
         )
+
+
+from cheetah.bmadx.interface import track_bmadx_quad
+
+class BmadQuadrupole(Element):
+    """
+    Test Bmad quadrupole element
+
+    :param length: Length in meters.
+    :param k1: Strength of the quadrupole in rad/m.
+    :param misalignment: Misalignment vector of the quadrupole in x- and y-directions.
+    :param tilt: Tilt angle of the quadrupole in x-y plane [rad]. pi/4 for
+        skew-quadrupole.
+    :param n_steps: Number of kicks in drift-kick-drift tracking.
+    :param name: Unique identifier of the element.
+    """
+
+    def __init__(
+        self,
+        length: Union[torch.Tensor, nn.Parameter],
+        k1: Optional[Union[torch.Tensor, nn.Parameter]] = None,
+        misalignment: Optional[Union[torch.Tensor, nn.Parameter]] = None,
+        tilt: Optional[Union[torch.Tensor, nn.Parameter]] = None,
+        num_steps: int = 10,
```

Comment:
Is this the only argument that's different between a Cheetah quad and a Bmad-X quad?